app/styles/app/modules/switch.sass

```diff
@@ -43,7 +45,9 @@
 
   .on
     display: none
-    margin-left: 0
+    margin-left: -49%
```

Comment:
@backspace This appears to have caused https://github.com/travis-ci/travis-ci/issues/8260. If I remove this, I get the correct display. Could you take a look?